src/core/flate_stream.js

```diff
@@ -148,6 +149,50 @@ class FlateStream extends DecodeStream {
     this.codeBuf = 0;
   }
 
+  get isAsync() {
+    return true;
+  }
```

Comment:
Please move this down, to maintain the same ordering of methods as in the `base_stream.js` file.